CFP.md

```diff
@@ -0,0 +1,63 @@
+CALL FOR PAPERS
+====
+
+--------------------------------------------------------------------------------
+*Workshop for NLP Open Source Software (NLP-OSS)*
+20 July 2018, Co-located with ACL 2018
+https://nlposs.github.io/
+
+Deadline for Long and Short Paper submission: 25th March 2018 
+(23:59, GMT-11)
+---------------------------------------------------------------------------------
+
+The First Workshop for NLP Open Source Software (NLP-OSS) will be co-located 
+with ACL 2018 at Melbourne, Australia on 20 July 2018. 
+
+The Workshop for NLP-OSS hopes to be the intellectual forum to collate engineering 
```

Comment:
I find this sentence verbose. And the "building aspect" negates the focus on OSS. Perhaps: "This workshop series will provide a forum to encourage and reflect upon the development of high quality open source infrastructure for NLP research, teaching and application."

Reply:
"it therefore focuses on social and engineering aspects of NLP rather than core science and technology."

Reply:
Although perhaps you don't agree entirely with that last statement

Reply:
I agree with @jnothman comments.

If we keep the initial version, I would slightly rephrase it:

> ... more on the building aspect of NLP **software** and less on scientific  novelty or state-of-art **models**.